applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -369,6 +369,61 @@ config BT_AUDIO_PACKING_INTERLEAVED
 	help
 	  ISO channels can either be interleaved or sequentially packed; sequential is the default one.
 
+config BT_AUDIO_PREFERRED_MIN_PRES_DLY_US
```

Comment:
I find it a tad strange that the pres delays are distributed over several files. Can we discuss?

Reply:
This is because the audio presentation delay is determined by the audio processing and is not related to Bluetooth. Then there is the Bluetooth preferred presentation delays that must be within these physical limitations. Hence the delay will appear in the two different modules.